samples/snippets/csharp/tour/arrays/arrays.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp1.0</TargetFramework>
+    <DebugType>portable</DebugType>
```

Comment:
sure, but when I create a brand new .NET Core app, I don't get this line which tells me that portable is probably the default value @blackdwarf?